.github/workflows/release.yml

```diff
@@ -37,12 +37,6 @@ jobs:
       run: |
         bundle config path vendor/bundle
         bundle install --jobs 4 --retry 3
-    - name: Update Tuist version
```

Comment:
this should not be deleted as the binary created will now have a wrong version.

Reply:
I undid all this so we can test releasing via action again as @pepicrft updated the token so it should 🤞🏼 work now